tests/datalab/issue_manager/test_null.py

```diff
@@ -20,7 +20,7 @@ def embeddings(self):
     def embeddings_with_null(self):
         np.random.seed(SEED)
         embeddings_array = np.random.random((4, 3))
-        embeddings_array[0][0] = np.NaN
+        embeddings_array[[0, 3], 0] = np.NaN
```

Comment:
Why is this array being changed?

Reply:
I see. The `rows_affected` were not computed correctly before, but now we can assert that they should be 0 and 3.